packages/rule-tester/src/RuleTester.ts

```diff
@@ -444,9 +444,10 @@ export class RuleTester extends TestFramework {
     return normalizedTests;
   }
 
-  defineRule(name: string, rule: AnyRuleModule): void {
+  defineRule(name: string, rule: Omit<AnyRuleModule, 'name'>): void {
```

Comment:
I don't think this really matters since it gets overwritten anyways 
```suggestion
  defineRule(name: string, rule: AnyRuleModule): void {
```